src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=1440589973&format=interactive",
+  sheets_gid="895329225",
+  sql_file="request_type_distribution_2021.sql"
+) }}
+
+Looking at total image bytes shows us that this metric has remained virtually unchanged from the previous year. This is likely due more and more images being served by content distribution networks (CDN), which apply strong optimizations to images as they are uploaded to their servers for use on future requests.
+
+## Conclusion
+
+The overall trend moving forward will likely continue to grow for desktop rendered pages and remain mostly unchanged for mobile. Just as they did, along with other browser vendors, for effort to move sites to HTTPS, Google has applied pressure on site owners to put their web pages on a diet, especially for mobile. Google's strategy has been solitified into a program where certain metrics (now called Core Web Vitals) that significantly impact the user experience reward web sites with smaller, faster loading pages by raising their rankings in search results. Conversely, site owners who continue to build heavy pages will be penalized in search results. In a world where small changes in search rankings can have a big impact on the success or failure of a site and their owner's, we should see greater emphasis on creating lighter web pages from design to hosted services that continue to find technological solutions to help keep page assets small and transported in a much faster way.
+
+But will the pressure to lighten the page load, so to speak, cross all web boundaries? What about web titans like Amazon will they follow suit? The answer is unclear. Predicting how the web will be shaped is difficult at best. But there is reason to be optimistic that both kings and subjects in the web world all have good reasons to to make lighter page weights a priority, even if the reasons for doing so differ. A small publisher on the web may be motivated to create ligher pages to compete in search ranking, increase the enjoyment of their users, and doing there part to reduce the amount of energy needed to store, transmit, and render those pages. The Amazon's may want to take advantage of reducing the size of page assets and services to reduce the spend required to serve those pages, and to move into newly emerging markets where users may not be able to buy super fast smartphones or have access to 5G data networks or high speed cable providers. Time will tell.
+
+What is clear is that the upside of lighter, faster loading web pages provide a strong incentive to commit to take advantage of emerging technologies, design and development techniques, and other solutions to stay ahead of the pack.
```

Comment:
This sentence does not make sense to me.

Reply:
Agreed